packages/cardano-services/test/StakePool/TypeormStakePoolProvider/util.test.ts

```diff
@@ -4,7 +4,7 @@ import { validateFuzzyOptions, withTextFilter } from '../../../src/StakePool/Typ
 describe('TypeormStakePoolProvider utils', () => {
   describe('validateFuzzyOptions', () => {
     it('throws if value is not a valid JSON encoded string', () =>
-      expect(() => validateFuzzyOptions('test')).toThrow('Unexpected token e in JSON at position 1'));
+      expect(() => validateFuzzyOptions('test')).toThrow(/Unexpected token/));
```

Comment:
This is an odd change for this PR, was this test flaky?

Reply:
Yes it was. It fails locally as it receives this error message "Unexpected token 'e', \"test\" is not valid JSON". Maybe updating the expected error message to match the received message would work better than regex? wdyt?